app/controllers/comment_liking_relations_controller.rb

```diff
@@ -23,7 +32,16 @@ def authenticate
       end
     end
 
+    def authorize
+      @user = User.find(comment_relation_params[:liking_user_id])
+      unless @user == current_user
+        flash[:warning] = 'You must log in to like.'
+        redirect_to root_url, status: :see_other
+      end
+    end
+
     def comment_relation_params
-      params.require(:comment_liking_relation).permit(:liked_comment_id)
+      params.require(:comment_liking_relation).permit(:liked_comment_id,
```

Comment:
no i znowu - czy to nie pojdzie z zapytania REST? Nie powinnismy miec user_id z authentykacji?

Reply:
Ech tak, dobra ostatecznie to przepisałem, by wykorzystywać tak jak mówisz. Myślałem wcześniej, że czasem jest to niezbędne, ale wcale nie jest.